.travis.yml

```diff
@@ -11,7 +11,7 @@ before_install:
     - chmod 777 ~/onFailure.sh
     - chmod 777 ~/onSuccess.sh
 install:
-    - mvn dependency:resolve dependency:resolve-plugins -DskipTests=true -Dmaven.javadoc.skip=true -B -V
+    - mvn clean install dependency:resolve dependency:resolve-plugins -DskipTests=true -Dmaven.javadoc.skip=true -B -V
```

Comment:
I suspect that is why this build worked for some PR's and not for others, that a host was reused to build artifacts and therefore were successful.  The artifacts have to be available.  If we want to get away from that we need to manually clean the `.m2` dir probably.  Or not do this step in the install phase and simply rely on the `install` goal to do all the dependency management.